ReactWindows/ReactNative.Shared/ReactRootView.cs

```diff
@@ -195,6 +196,28 @@ protected override Size MeasureOverride(Size availableSize)
             return result;
         }
 
+#if WINDOWS_UWP
+        /// <summary>
+        /// Override ensuring the creation of an AutomationPeer for ReactRootView 
+        /// root view.
+        /// </summary>
+        /// <returns>The new AutomationPeer.</returns>
+        protected override AutomationPeer OnCreateAutomationPeer()
+        {
+            //
+            // Older Windows OS'es (RS1, for ex.) don't create automation peers for normal Canvas elements.
+            // This is not an issue in general, but for ReactRootView in particular is, since it is the root of accessibility traversal
+            // done in AccessibilityHelper and so it needs a peer.
+            var peer = base.OnCreateAutomationPeer();
+            if (peer == null)
+            {
+                // Fallback to creating peer if base didn't provide one.
+                peer = new FrameworkElementAutomationPeer(this);
```

Comment:
The automation peer that is being created should match the type of the element. What is the xaml element represented by ReactRootView? Because FrameworkElementAutomationPeer is the base for many other types of more specialized automationpeers. If ReactRootView XAML element is more specialized, the automation peer created for it must match it

Reply:
ReactRootView is a Canvas. RS1 creates no peer for this whatsoever. 
More, the app doesn't set and accessibility properties on this element.

So I don't think bad of returning a simple peer here. It's more than what Canvas offers (nothing), and good enough to ensure the root of the tree is in the root view (se we can traverse).


---
In reply to: [178354815](https://github.com/Microsoft/react-native-windows/pull/1748#discussion_r178354815) [](ancestors = 178354815)